manifests/cf-manifest/cloud-config/020-cf-vm-types.yml

```diff
@@ -0,0 +1,125 @@
+---
+meta:
+  default_env:
+    bosh:
+      password: (( grab secrets.vcap_password ))
+
+vm_types:
+  - name: small
+    network: cf
+    env: (( grab meta.default_env ))
+    cloud_properties:
+      instance_type: m3.medium
```

Comment:
Shall we than remove this block and move all references to `small` type to medium? It's a little bit confusing now.